src/docs/development/tools/devtools/code-size.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Using the Code Size Tool
+description: Learn how to use the DevTools code size tool.
+---
+
+## What is it?
+
+The code size tool allows you to visualize, analyze, and diff code / app size
+data.
+
+## How to use it
+
+If DevTools is already connected to a running application, navigate to the
+"Code Size" tab.
+
+![Screenshot of code size tab]({% asset tools/devtools/code_size_tab.png @path %})
+
+If DevTools is not connected to a running application, you can
+access the tool from the landing page that appears once you have launched
```

Comment:
Is it clear to all readers how to launch DevTools without being attached to a running application?